XM_Dinner/201_Unown_I.yaml

```diff
@@ -0,0 +1,22 @@
+species: Unown

```

Comment:
Looks pretty weak, but that may be me under-compensating for milk drink. Thoughts?

Reply:
Heat Wave and Crunch are p solid especially with their secondary effects, plus Sweet Scent can make sure they miss less. Dinner's balancing is p confusing anyway lol.

Reply:
sweet scent is basically a filler move for thematic reasons, crunch is 100 acc and heat wave 90, it's not worth taking a turn to sweet scent in normal circumstances, and the secondary effects aren't too noticeable (10% burn and 20% def drop).  The main thing with this set is that unown has low bulk, mediocre offenses and this set has no STAB. heat wave comes out at 702,000 power which is below curve while crunch is 700,800 and it's bulk is only 34,848. Milk drink makes up for a lot but it still seems wimpy to me, but I'll let other more experienced setbuilders chip in